CacheMultimap.adoc

```diff
@@ -46,33 +45,52 @@ public interface CacheMultimap<K, V> {
 
 The underlying implementation will wrap a normal Cache. Some of the suggestions :
 
-== Core Module
 
-[source,java]
-----
-public interface EmbeddedCacheManager {
- ...
- 
- <K, V> CacheMultimap<K, V> getCacheMultimap(String cacheName, boolean createIfAbsent);
-}
+Apparently there is a problem with functional commands.
+They don't really work efficiently over Hot Rod (does get/replace in a loop).
+We would need to add some more handling in the protocol to allow for only partial replication
+of values and only 1 remote call.
 
-----
 
-== Commons Module
+== Embedded Multimap
 
-[source, java]
-----
-public class CacheMultimapFactory {
+A new maven module is created.
 
-   static <K, V> CacheMultimap<K, V> CacheMultimapFactory.get(Cache<K,Object> c) {
-      ...
-   }
-}
-----
+To create a multimap cache
+
+```java
+      EmbeddedCacheManager cm = ...
+      MultimapCacheManager multimapCacheManager = EmbeddedMultimapCacheManagerFactory.from(cm);
```

Comment:
@rvansa I was thinking that we might prefix cache names and config with multimap. I didn't think too much about it yet because this is the first version and I'm planning to mark it as experimental.